vagrant_config.json

```diff
@@ -139,9 +139,20 @@
           "action": "add" // one of add, remove. Default: add.
```

Comment:
I see the point due to the way we introduce "_a_" and "_ua_", let's make it consistent for both config_paths and extension_paths, I'll update extensions_path behavior after when this is merged.